Src/PChecker/CheckerCore/SystematicTesting/TestingEngine.cs

```diff
@@ -783,7 +783,7 @@ private void ConstructReproducableTrace(ControlledRuntime runtime)
                 var step = runtime.Scheduler.ScheduleTrace[idx];
                 if (step.Type == ScheduleStepType.SchedulingChoice)
                 {
-                    stringBuilder.Append($"({step.ScheduledOperationId})");
+                    stringBuilder.Append($"{step.ScheduledOperationName}");
```

Comment:
I dont understand this.

Reply:
Same as last comment.

Reply:
Reverted via https://github.com/p-org/P/pull/614/commits/60506120fcd3a6e626511a8934e530f5550b0a87